extensions/lifecycle/LifecycleConfigValidator.js

```diff
@@ -51,6 +51,7 @@ const joiSchema = joi.object({
         // the processing, no need to add more here to avoid
         // overloading the system
         concurrency: joi.number().greater(0).default(1),
+        maxQueued: joi.number().greater(0).default(1000),
```

Comment:
Should we make the default consistent?

```suggestion
        maxQueued: joi.number().greater(0).default(MAX_QUEUED_DEFAULT),
```